src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -53,6 +53,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="System.Memory" Condition="'$(TargetFramework)' == '$(NETFXTargetFramework)'" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
I first tried with a condition that excluded .NETCoreApp TFMs, but this project builds with net5.0, and STJ in .NET 5 doesn't have a `Deserialize<T>(Stream)` overload.

Reply:
Messaged offline.

Reply:
I'll be able to re-add the TargetFrameworkIdentifier != .NETCoreApp condition once this gets merged:
* https://github.com/NuGet/NuGet.Client/pull/6086